beacon-chain/db/kv/checkpoint.go

```diff
@@ -83,7 +83,7 @@ func (s *Store) SaveFinalizedCheckpoint(ctx context.Context, checkpoint *ethpb.C
 		hasStateSummary := s.hasStateSummaryBytes(tx, bytesutil.ToBytes32(checkpoint.Root))
 		hasStateInDB := tx.Bucket(stateBucket).Get(checkpoint.Root) != nil
 		if !(hasStateInDB || hasStateSummary) {
-			return errMissingStateForCheckpoint
+			return errors.Wrapf(errMissingStateForCheckpoint, "could not save Finalized checkpoint, finalized root: %#x", bytesutil.Trunc(checkpoint.Root))
```

Comment:
Why are you truncating to first 6 bytes?

Reply:
I just dislike the full log when reading logs... I gotta be very unlucky to hit the same root many times in a debug session